avaframe/tests/test_DFAfunctionsCython.py

```diff
@@ -235,7 +235,7 @@ def test_reprojectionC(capfd):
 
     for reprojectionIterations in range(10):
         xpn, ypn, iCell, Lx0, Ly0, w0, w1, w2, w3 = DFAfunC.samosProjectionIteratrive(x1, y1, z1,
-                        ZZ, Nx, Ny, Nz, csz, ncols, nrows, interpOption, reprojectionIterations)
+                                                                                      ZZ, Nx, Ny, Nz, csz, ncols, nrows, interpOption, reprojectionIterations)
```

Comment:
For this (and the line lower down) I'd prefer to rather violate the indent pep but not have such long lines...

Reply:
atom did this automatically, I ll revert it